### PR TITLE
docs: add docs for using an openai complaint model endpoint

### PR DESCRIPTION
**Pull request by langchain-infra**

### CODE REVIEW

File: docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 4
+---
+
+# Use a custom OpenAI compliant model
```
Comment (jakerachleff) [inline docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx:5]:
```suggestion
# Run the playground against your own AI system using an OpenAI compliant format
```

File: docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+The LangSmith playground allows you to use any model that is compliant with the OpenAI API. You can utilize your model by setting the Proxy Provider for `OpenAI` in the playground.
```
Comment (jakerachleff) [inline docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx:7]: We should break this into two sections. 

Step 1: accept openai compliant format in your model or system. This just links to the docs of what they need to set up
Step 2: the instructions in playground

File: docs/prompt_engineering/how_to_guides/index.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Quickly iterate on prompts and models in the LangSmith Playground.
 - [Use custom TLS certificates](./how_to_guides/playground/custom_tls_certificates)
 - [Use a custom model](./how_to_guides/playground/custom_endpoint)
 - [Save settings configuration](./how_to_guides/playground/save_model_configuration)
+- [Use a custom OpenAI compliant model](./how_to_guides/playground/custom_openai_compliant_model)
```
Comment (jakerachleff) [inline docs/prompt_engineering/how_to_guides/index.md:21]: change this too to match new title

File: docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx (hunk 1 above)
Comment (langchain-infra) [reply docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx]: fixed

File: docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx (hunk 2 above)
Comment (langchain-infra) [reply docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx:7]: fixed

File: docs/prompt_engineering/how_to_guides/index.md (hunk 3 above)
Comment (langchain-infra) [reply docs/prompt_engineering/how_to_guides/index.md]: fixed

**Review by agola11 — APPROVED** [review]

**Pull request merged**